herddb-core/src/main/java/herddb/model/DataConsistencyStatementResult.java

```diff
@@ -0,0 +1,31 @@
+/*
+ Licensed to Diennea S.r.l. under one
+ or more contributor license agreements. See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership. Diennea S.r.l. licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+
+ */
+package herddb.model;
+
+/**
+ *
+ * @author Hamado.Dene
+ */
+public class DataConsistencyStatementResult extends StatementExecutionResult {
+
+    public DataConsistencyStatementResult(long transactionId) {
```

Comment:
you don't need transactionId, there is not trasaction.
it is bery usefull to have:
- boolean ok -> success/failure
- String message - > additional message, expecially for error

Reply:
In this case, I don't need to extend StatementExecutionResult right?

I have to manage  differently TableConsistencyCheckStatement e  TableSpaceConsistencyCheckStatement in DbManager and TableSpaceManager.
I can't manage them in executeStatementAsync because they return a CompletableFuture<StatementExcutionResult>. 
Do you have some suggestions?

Reply:
Every execution of *executeStatementAsync* must return a StatementExecutionResult.
We wrap it with *CompletableFuture* because most of the code is asynchronous.

Reply:
You can wrap it with CompletableFuture.completedFuture